app/main.py

```diff
@@ -2,24 +2,84 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: type, name: str) -> None:
+        self.name = name
+
+    def __get__(self,
+                instance: "SlideLimitationValidator",
+                owner: type
+                ) -> tuple:
+        return getattr(instance, self.name)
+
+    def __set__(self,
+                instance: "SlideLimitationValidator",
+                value: int
+                ) -> None | bool:
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError
+        return True
```

Comment:
Use `settatr` here, do not `return True`. Change the corresponding type annotation